GeneticDFA/DFAChromosome.cs

```diff
@@ -0,0 +1,128 @@
+﻿using GeneticSharp;
+
+namespace GeneticDFA;
+
+public class DFAChromosome : IChromosome
+{
+    public List<DFAStateModel> States { get; } = new List<DFAStateModel>();
+    public List<DFAEdgeModel> Edges { get; } = new List<DFAEdgeModel>();
+    public List<DFAEdgeModel> NonDeterministicEdges { get; private set; } = new List<DFAEdgeModel>();
+    public DFAStateModel StartState { get; }
+    public double? Fitness { get; set; }
+    public int Size => States.Count + Edges.Count;
+    private int _nextStateID = 0;
+    private int _nextEdgeID = 0;
+
+    public DFAChromosome(List<DFAStateModel> states, List<DFAEdgeModel> edges, DFAStateModel startState)
+    {
+        States = states;
+        Edges = edges;
+        StartState = startState;
+    }
+
+    public DFAChromosome()
+    {
+        
+    }
+
+
+    public IChromosome CreateNew()
+    {
+        return new DFAChromosome();
+    }
+
+    public void FindAndAssignNonDeterministicEdges()
```

Comment:
Jeg føler den her funktion hører til i fitness og ikke på chromosome. Det ville også gøre sådan, at du ikke behøver typecast i evaluate.

Reply:
Will be moved to mutation